beard.js

```diff
@@ -97,6 +99,7 @@ module.exports = function(opts = {}) {
       .replace(exps.includeFn, parse.includeFn)
       .replace(exps.block, parse.block)
       .replace(exps.blockEnd, parse.blockEnd)
+      .replace(exps.encode, parse.encode)
```

Comment:
@joeosburn are we sure this is where we want this to run? IOW, if we have an encode inside a block will there be any chance of a parse error?